src/milia/api/project.cljc

```diff
@@ -87,15 +87,22 @@
 
 (defn add-star
   "Add star to project for this user."
-  [id]
+  [id & {:keys [callback]}]
   (let [url (make-url "projects" id "star")]
-    (parse-http :post url)))
+    (parse-http :post url :callback callback)))
 
 (defn remove-star
   "Remove star from project for this user."
-  [id]
+  [id & {:keys [callback]}]
   (let [url (make-url "projects" id "star")]
-    (parse-http :delete url)))
+    (parse-http :delete url :callback callback)))
+
+(defn toggle-star
+  "Toggle between starred and unstarred for a user's project"
+  [id star? callback]
+    (if star?
```

Comment:
you can do this more concisely by pushing the if inside to varying the function called but with a single argument list, like

```clojure
((if star? add-star remove-star) id ...)
```